instructure/canvas-ios
# Share extension dashboard notification UI

Dashboard notification card for showing file upload progress. View Model is just a skeleton for now. 

refs: MBL-16146
affects: Student
release note: Dashboard notification card view for file upload progress

test plan: 
This PR is scoped down to the UI component only. 
In order to see the component: 
1. Open `FileUploadNotificationCardListtViewModel`
2. Delete line **34** & **47**

<br>

## Screenshots 
<table>
 <tr>
  <td>Dark mode</td>
  <td>Light mode</td>
 </tr>
  <tr>
  <td><img src="https://user-images.githubusercontent.com/110813321/185936842-797f567f-7322-4215-ac36-068824fc17e6.PNG" height=500></td>
  <td><img src="https://user-images.githubusercontent.com/110813321/185936875-8b9c67d3-feaf-4aa8-97e8-7c8db46bc957.PNG" height=500></td>
 </tr>
</table>
